DigitalOffice/Services/ProjectService/Repositories/GetProjectInfoByIdRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using ProjectService.Database;
+using ProjectService.Database.Entities;
+using ProjectService.Repositories.Interfaces;
+using System;
+using System.Linq;
+
+namespace ProjectService.Repositories
+{
+    public class GetProjectInfoByIdRepository : IGetProjectInfoByIdRepository
+    {
```

Comment:
do we need separate repositories for get, post, ….

Reply:
Due to this, we won't have classes with lots of extra inherited functions from the interface, I think

Reply:
But I think, that you have to rename it.

Reply:
fixed